website/docs/patterns/validation-form/examples/form-validation-example-by-`unfocus`.tsx

```diff
@@ -175,10 +175,15 @@ class Demo extends React.Component {
 
                     return (
                       <Tooltip>
-                        <Tooltip.Popper id='form-select-error' theme='warning' visible={showError}>
+                        <Tooltip.Popper
+                          id='form-select-error'
+                          theme='warning'
+                          placement='top'
+                          visible={showError}
+                        >
                           {meta.error}
                         </Tooltip.Popper>
-                        <Tooltip.Trigger w={'100%'}>
+                        <Tooltip.Trigger w={'100%'} inline={false} {...input}>
```

Comment:
I think, you could try to use something like `<Select.Trigger tag={Tooltip.Trigger}..../>`

Reply:
I'm not sure why, but specifying TooltipTrigger as tag here does not work.

Reply:
But in this solution, I think we will have problems with accessibility... because your trigger is just a div without any role